google/ggrc-core
# GGRC-7306 Update review/review display for Snapshots

# Issue description
Extend review / review display status for Snaphots with Risk type

# Steps to test the changes
Run integration tests
Open Snapshots for early created Risks and check review status - status should be showed 

# Solution description
Extend populate snapshot display method with Risk type

# Sanity checklist


# PR Review checklist
